advisories/a48860.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Technical Advisory 48860
+summary: Data corruption/loss issue with snapshots and delete range
+toc: true
+---
+
+Publication date: May 18, 2020.
+
+## Description
+
+Range deletions (as caused by certain [table drops](../v20.1/drop-table.html), [table truncates](../v20.1/truncate.html), and certain rebalancing operations during [node decommissioning](../v20.1/remove-nodes.html) or failover), when occurring concurrently with replica [garbage collection](../v20.1/architecture/storage-layer.html#garbage-collection), can yield permanent data corruption or loss in rare cases.
+
+## Statement
+
+A defect in RocksDB allowed it to violate its guarantees for snapshots, causing corruption under specific and extremely rare circumstances. This corruption is irrecoverable using regular node failover and must be addressed by restoring a backup when it occurs.
+
+This bug has been fixed in versions v19.1.9, v19.2.7, and v20.1.0. All users are encouraged to [upgrade](../v20.1/upgrade-cockroach-version.html).
```

Comment:
should we encourage upgrades based on the target release? Otherwise we are promoting v20.1 upgrade docs.

Reply:
Fixed.